scripts/dotnet45.ps1

```diff
@@ -0,0 +1,8 @@
+iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/mwrock/boxstarter/master/BuildScripts/bootstrapper.ps1'))
```

Comment:
I dont think this is necessary. See coment above. Did you have issues running outside of a task?